client/src/endpoint/asset.rs

```diff
@@ -0,0 +1,190 @@
+use error::Result;
+use std::str::FromStr;
+use stellar_resources::Asset;
+use super::{EndPoint, Order, Records};
+use http::{Request, Uri};
+
+/// Represents the all assets end point for the stellar horizon server. The endpoint
+/// will return all assets filtered by a myriad of different query params.
+///
+/// https://www.stellar.org/developers/horizon/reference/endpoints/assets-all.html
+///
+/// ## Examples
+///
+/// #### Asset code
+///
+/// Fetches all records for a given asset code.
+///
+/// ```
+/// # use stellar_client::sync::Client;
+/// # use stellar_client::endpoint::AllAssets;
+/// #
+/// let client      = Client::horizon_test().unwrap();
+/// let endpoint    = AllAssets::default().asset_code("USD");
+/// let records     = client.request(endpoint).unwrap();
+/// #
+/// # assert!(records.records().len() > 0);
+/// # assert_eq!(records.records()[0].code(), "USD");
+/// ```
+///
+/// #### Issuer
+///
+/// Fetches all records for a given asset issuer.
+///
+/// ```
+/// # use stellar_client::sync::Client;
+/// # use stellar_client::endpoint::AllAssets;
+/// #
+/// let client = Client::horizon_test().unwrap();
+/// let endpoint = AllAssets::default()
+///     .asset_issuer("GA2LJYWSEZWZYIPMM4G6JXHM3PRHIEAD2TIR4IVJFZIAECYO7RDQJWOM");
```

Comment:
They occasionally reset the testnet https://www.stellar.org/developers/guides/concepts/test-net.html

Reply:
grrr

Reply:
It is an account from the examples but yea, i don't want a flaky test.